README.md

```diff
@@ -51,7 +51,7 @@ const oapi = openapi({
 })
 
 // This will serve the generated json document(s)
-// (as well as swatter-ui or redoc if configured)
```

Comment:
lol, not sure, I kind of want to keep this one

Reply:
Up to you ;)